api/about.json

```diff
@@ -0,0 +1,9 @@
+{
+ "0.01": [
+   {
+    "logo": "http://bit.ly/2niBg5Y",
+    "website":"https://pydelhi.org/",
+    "details":"PyDelhi is a volunteer organization,group of technically inclined students, professionals, teachers and entrepreneurs who want to learn something new having a common primary interest in the python programming language." 
```

Comment:
space after `organization,` and it should be `Python` instead of python IMHO.

Also, we should have common JSON for both mobile and website. Replicating stuff would make it difficult to maintain and update.